packages/flutter/test/material/navigation_rail_test.dart

```diff
@@ -5423,43 +5479,56 @@ TestSemantics _expectedSemantics() {
   return TestSemantics.root(
     children: <TestSemantics>[
       TestSemantics(
+        id: 1,
```

Comment:
Can you please omit the IDs from this test like it was before? They are not relevant for the test and just make it more brittle if the ID assignment algorithm changes.

Reply:
Deleted.